3-async-await/solution.js

```diff
@@ -20,16 +20,43 @@ Example:
 9. as extra challenge: add Promise.race() and Promise.any(), and try to get the idea of what happens
 */
 
-function solution() {
+const productMethod = require('./products')
+const priceMethod = require('./prices')
```

Comment:
Same suggestion here. Use descriptive variables/function names.
```suggestion
const getProducts = require('./products')
const getPrices = require('./prices')
```